content/applications/finance/accounting/payables/supplier_bills/ocr.rst

```diff
@@ -3,13 +3,17 @@ Digitize Vendor Bills with Optical Character Recognition (OCR)
 ==============================================================
 
 Encoding bills manually can be a time-consuming task. Having a solution that allows you to digitize
-them and automatically import the data to your database reduces errors and saves you time.
+them and automatically import the data into your database reduces errors and saves you time.
 
+.. important::
+   - Try it out! Upload one of your Bills or try one of our samples.
+   - Click `here <https://www.odoo.com/app/invoice-automation>`_ to evaluate it now!
```

Comment:
```suggestion
   `Try it out <https://www.odoo.com/app/invoice-automation>`_! Upload one of your Bills or try one of our samples.
```
IMHO, a bulleted list is used to highlight different points, here both points share the same meaning/logic.